### PR TITLE
Update SpreadsheetExport.php

### PR DESCRIPTION
**Pull request by funkycram**
Remove pagination if `$dataProvider` is an instance of `ArrayDataProvider`

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [ ] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: protected/humhub/components/export/SpreadsheetExport.php [hunk 1]
```diff
@@ -136,7 +137,7 @@ public function init()
             ]);
         }
 
-        if ($this->dataProvider instanceof ActiveDataProvider) {
+        if ($this->dataProvider instanceof ActiveDataProvider || $this->dataProvider instanceof ArrayDataProvider) {
```
Comment (luke-) [inline protected/humhub/components/export/SpreadsheetExport.php:140]: Just `if ($this->dataProvider instanceof BaseDataProvider) {` should be correct here.

**Review by luke- — CHANGES_REQUESTED** [review]
Thanks, can you please chekc my comment

**Comment by funkycram** [comment]
Thanks @luke-, yes, it is better to check if it's an instance of the parent class containing the `setPagination` method.

**Review by luke- — APPROVED** [review]

**Comment by luke-** [comment]
@funkycram Thanks, it's merged now!

**Pull request merged**